src/QMCDrivers/SimpleFixedNodeBranch.cpp

```diff
@@ -743,11 +743,18 @@ int SimpleFixedNodeBranch::resetRun(xmlNodePtr cur)
     std::string reconfig("no");
     // method is actually IndexType so conceivably indicates much more that reconfig="yes" or "no"
     if (WalkerController->get_method())
-      reconfig = "yes";
+      reconfig = "runwhileincorrect"; // forces SR during warmup?
     std::string reconfig_prev(reconfig);
+    std::string reconfig_read = "no";
     ParameterSet p;
-    p.add(reconfig, "reconfiguration", "string");
+    p.add(reconfig_read, "reconfiguration", "string");
     p.put(cur);
+    if(reconfig_read != "no" && reconfig_read != "runwhileincorrect")
+    {
+      reconfig = reconfig_read;
+      // remove this once bug is fixed
+      APP_ABORT("Reconfiguration is currently broken.  Please try again without reconfiguration.")
+    }
```

Comment:
There seem to be logic changes.
In the old code, the default value of reconfig depends on the existing WalkerController. Then the XML parsing overwrites reconfig if an input entry exists. Now we just need to check if reconfig is neither "no" nor "runwhileincorrect" and abort.
The changed code seems incorrect. If there is no input or input value is "no" or "runwhileincorrect", reconfig_read result gets ignored. I think if we replace reconfig_read with reconfig, the code should work for our needs.

Reply:
Done.